Cargo.toml

```diff
@@ -5,7 +5,8 @@ panic = 'unwind'
 members = [
     'node',
     'runtime',
-    "pallets/pdex-migration"
+    'pallets/pdex-migration',
+    'pallets/Polkadex_Ido'
```

Comment:
Is there any reason why not to align to naming convention that is already there in pallets when naming new one?